presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -273,6 +272,7 @@ primaryExpression
     | identifier '->' expression                                                     #lambda
     | '(' identifier (',' identifier)* ')' '->' expression                           #lambda
     | '(' query ')'                                                                  #subqueryExpression
+    | EXISTS '(' query ')'                                                           #exists
```

Comment:
For example `SELECT EXISTS(SELECT true) = EXISTS(SELECT true)` didn't work.

This queries are already added to `AbstractTestQueries`. Would you like to have additional tests for them in `TestSqlParser`?